src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -536,6 +577,18 @@ class ElytraFlight : Module() {
         }
     }
 
+    fun updatePathfindState() {
+        val autowalk = MODULE_MANAGER.getModuleT(AutoWalk::class.java)
+        pathfind = autowalk.mode.value == AutoWalk.AutoWalkMode.PATHFIND && autowalk.isEnabled
+        if (mc.player != null) {
+            if (pathfind && !Astarpathfinder.enabled) {
+                Astarpathfinder.init()
+            } else if (!pathfind && Astarpathfinder.enabled) {
+                Astarpathfinder.turnOff()
+            }
+        }
```

Comment:
Maybe this should *all* be in the if mc.player != null statement

Reply:
wouldn't hurt  i guess